Formula/jigdo.rb

```diff
@@ -3,10 +3,16 @@
 # tool that has been on life support and still works. Only build the CLI tool.
 class Jigdo < Formula
   desc "Tool to distribute very large files over the internet"
-  homepage "http://atterer.org/jigdo/"
+  homepage "https://www.einval.com/~steve/software/jigdo/"
```

Comment:
Updated homepage and livecheck to reference maintained fork (linked from previous homepage); however, updating to v0.8+ needs [`docbook-utils`](https://repology.org/project/docbook-utils/versions) to build. So, will leave update for later.